cluster/gke/util.sh

```diff
@@ -165,10 +165,12 @@ function kube-up() {
 #   NODE_TAG
 function test-setup() {
   echo "... in gke:test-setup()" >&2
+  KUBE_TEST_DEBUG=y "${KUBE_ROOT}/cluster/kube-up.sh"
+
   # Detect the project into $PROJECT if it isn't set
   detect-project >&2
```

Comment:
I guess detect-project isn't really necessary, since it was first in gce's `test-setup` but not first here. feels like maybe it should be more consistent, though?